### PR TITLE
Init UI tab List Photos

### PR DESCRIPTION
**Pull request by thaohq-0814**
Init UI and events tab List Photos
Link: https://edu-redmine.framgia.vn/issues/6622
image

### CODE REVIEW

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/java/com/framgia/my_editor_02/utils/EndlessScrollListener.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    private static final int DEFAULT_VALUE = 0;
+    private int mVisibleThreshold;
+    private int mCurrentPage;
+    private int mPreviousTotalItemCount;
+    private boolean mIsLoading;
+
+    private RecyclerView.LayoutManager mLayoutManager;
+
+    public EndlessScrollListener(LinearLayoutManager layoutManager) {
+        this.mLayoutManager = layoutManager;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/utils/EndlessScrollListener.java]: remove this

File: app/src/main/java/com/framgia/my_editor_02/utils/EndlessScrollListener.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        mVisibleThreshold = DEFAULT_VISIBLE_THRESHOLD;
+        mCurrentPage = DEFAULT_CURRENT_PAGE;
+    }
+
+    public EndlessScrollListener(GridLayoutManager layoutManager) {
+        this.mLayoutManager = layoutManager;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/utils/EndlessScrollListener.java]: remove this

File: app/src/main/java/com/framgia/my_editor_02/utils/OnItemRecyclerViewClick.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.framgia.my_editor_02.utils;
+
+public interface OnItemRecyclerViewClick<T> {
+    void onClick(T item);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/utils/OnItemRecyclerViewClick.java]:
```suggestion
    void onItemClick(T item);
```

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**